### PR TITLE
The magboots are actually correct now.

### PR DESCRIPTION
**Pull request by remona-minett**
We finally got it :1st_place_medal:

### CODE REVIEW

**Comment by Nabski89** [comment]
These are still supposed to be the same item tier so if you are swapping the precision swap the health too.

**Comment by remona-minett** [comment]
Got it.

File: src/data/items/slotLimb.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ export default {
 		equipmentSlot: "limb",
 		overlay: require("@/assets/art/combat/items/limb/shoe_black_overlay.png"),
 		stats: {
-			maxHealth: 60,
+			maxHealth: 120,
```
Comment (Nabski89) [inline src/data/items/slotLimb.js:164]: So close

File: src/data/items/slotLimb.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ export default {
 		overlay: require("@/assets/art/combat/items/limb/shoe_magboots_overlay.png"),
 		stats: {
 			maxHealth: 60,
```
Comment (Nabski89) [inline src/data/items/slotLimb.js:147]: Here

**Comment by Nabski89** [comment]
You did the black shoes.

**Comment by remona-minett** [comment]
good lord, im fixing it rn

**Pull request closed without merging**